Cargo.toml

```diff
@@ -1,20 +1,43 @@
+
 [package]
 name = "mavlink"
 version = "0.4.2"
-authors = ["Tim Ryan"]
+authors = ["Todd Stellanova", "Michal Podhradsky", "Kevin Mehall", "Tim Ryan"]
 build = "build/main.rs"
-description = "Parses the MAVLink data interchange format for UAVs."
+description = "Implements the MAVLink data interchange format for UAVs."
 license = "MIT/Apache-2.0"
-repository = "https://github.com/3drobotics/rust-mavlink"
+repository = "https://github.com/tstellanova/rust-mavlink.git"
 
 [build-dependencies]
 crc16 = "0.3.3"
-byteorder = "0.5.3"
+bytes = "0.4"
 xml-rs = "0.2"
+quote = "0.3"
```

Comment:
Be advised that quote 0.3 is two years old and differs significantly from 0.6.

You could make an argument that 0.3 is better for this use case because it doesn't need to integrate with proc-macro and lets you get away with abusing `Ident` into a way of pasting in arbitrary strings, but know that it's outdated and unmaintained.

Reply:
@podhrmic I forget whether I introduced this in an effort to move to stable.  Do you want to tackle this?